### PR TITLE
CORE-532: Updated Unit Tests

### PR DESCRIPTION
**Pull request by EBGToo**
Includes a fix in Swift and Java to the `supportedAddressSchemes` map to use 'bitcoincash-testnet' (not 'bitcoin-cash-testnet')

The test `testsTransferBTC_P2P()` fails; captured as CORE-546

The code adds in the BRD token on Testnet; needed in Java

There is a wallet name test that uses "eth" and "BRD"; someday "eth" will be "ETH".

### CODE REVIEW

File: Java/CoreCrypto/src/main/java/com/breadwallet/corecrypto/System.java [hunk 1]
```diff
@@ -248,11 +248,11 @@
     static {
         ImmutableMap.Builder<String, WalletManagerMode> builder = new ImmutableMap.Builder<>();
         builder.put("bitcoin-mainnet", WalletManagerMode.P2P_ONLY);
-        builder.put("bitcoin-cash-mainnet", WalletManagerMode.P2P_ONLY);
+        builder.put("bitcoincash-mainnet", WalletManagerMode.P2P_ONLY);
```
Comment (michael-brd) [inline Java/CoreCrypto/src/main/java/com/breadwallet/corecrypto/System.java]: I think this change is already present in crypto. How is it showing as a diff?

File: Swift/BRCrypto/BRCryptoSystem.swift [hunk 2]
```diff
@@ -124,10 +124,10 @@ public final class System {
                          (name: "Gwei",  code: "gwei", decimals:  9, symbol: BlockChainDB.Model.lookupSymbol ("gwei")),
                          (name: "Ether", code: "eth",  decimals: 18, symbol: BlockChainDB.Model.lookupSymbol ("eth"))]),
 
-//        (id: "BRD Token Testnet", name: "BRD Token", code: "brd", type: "erc20", blockchainID: "ethereum-ropsten",
-//         address: BlockChainDB.Model.addressBRDTestnet, verified: true,
-//         demoninations: [(name: "BRD_INTEGER",   code: "BRDI",  decimals:  0, symbol: "brdi"),
-//                         (name: "BRD",           code: "BRD",   decimals: 18, symbol: "brd")]),
+        (id: "ethereum-ropsten:0x7108ca7c4718efa810457f228305c9c71390931a", name: "BRD Token Testnet", code: "BRD", type: "erc20", blockchainID: "ethereum-ropsten",
```
Comment (michael-brd) [inline Swift/BRCrypto/BRCryptoSystem.swift]: Should this be mirrored in Java now that it is uncommented? I can do it, if so.

Comment (EBGToo) [reply Swift/BRCrypto/BRCryptoSystem.swift]: I think yes; just for consistency.

File: Java/CoreCrypto/src/main/java/com/breadwallet/corecrypto/System.java [hunk 3]
```diff
@@ -167,8 +166,10 @@
                     ImmutableList.of(CurrencyDenomination.SATOSHI, CurrencyDenomination.BCH_BITCOIN_TESTNET)),
 
             new com.breadwallet.crypto.blockchaindb.models.bdb.Currency("ethereum-ropsten:__native__", "Ethereum Testnet", "eth", "native", "ethereum-ropsten", null, true,
-                    ImmutableList.of(CurrencyDenomination.ETH_WEI, CurrencyDenomination.ETH_GWEI,
-                            CurrencyDenomination.ETH_ETHER))
+                    ImmutableList.of(CurrencyDenomination.ETH_WEI, CurrencyDenomination.ETH_GWEI, CurrencyDenomination.ETH_ETHER)),
+
+            new com.breadwallet.crypto.blockchaindb.models.bdb.Currency("ethereum-ropsten:0x7108ca7c4718efa810457f228305c9c71390931a", "BRD Token Testnet", "BRD", "erc20", "ethereum-ropsten", ADDRESS_BRD_MAINNET, true,
+                    ImmutableList.of(CurrencyDenomination.BRD_INT, CurrencyDenomination.BRD_BRD))
```
Comment (EBGToo) [inline Java/CoreCrypto/src/main/java/com/breadwallet/corecrypto/System.java]: This should be `ADDRESS_BRD_TESTNET` (the 0x7108... one).

Comment (michael-brd) [reply Java/CoreCrypto/src/main/java/com/breadwallet/corecrypto/System.java]: Fixed with force push.

**Pull request merged**